packages/vite/src/node/ssr/runtime/runtime.ts

```diff
@@ -266,7 +265,7 @@ export class ViteRuntime {
     this.debug?.('[vite-runtime] fetching', id)
     // fast return for established externalized patterns
     const fetchedModule = id.startsWith('data:')
-      ? ({ externalize: id, type: 'builtin' } as FetchResult)
+      ? ({ externalize: id, type: 'builtin' } satisfies FetchResult)
```

Comment:
This `as` is not needed. Instead of `satisfies`, `{ externalize: id, type: 'builtin' as const }` can be used as well. But I chose `satisfies` because I thought that would represent the intent more accurately.